app/services/migration/migrators/user.rb

```diff
@@ -33,10 +33,14 @@ def call
         trns = unique_validated_trns(ecf_user)
         validate_multiple_trns!(trns, user)
 
+        emails = unique_emails(ecf_user)
+        validate_multiple_emails!(emails, user)
+        validate_existing_email_match!(emails.first, user)
+
         user.update!(
           trn: ecf_user.teacher_profile&.trn ? ecf_user.teacher_profile.trn : trns.last,
           full_name: ecf_user.full_name || user.full_name,
-          email: ecf_user.email || user.email,
+          email: emails.first || user.email,
           uid: ecf_user.get_an_identity_id || user.uid,
```

Comment:
thinking about DoB/nino/active alert, can we pull those back from ECF, although they don't get updated by admin I wonder if someone went into the console and updated, I think worth overwriting from ECF anyway?

Reply:
could we also pull in `trn_verified` as `teacher_reference_number_verified` from ECF, as that could be updated on the ECF side as well